tools/socket_passing.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python2.7
```

Comment:
I don't think everyone will be running precisely Python 2.7 or have Python in this location. Why the change?